ext/dcos-installer/dcos_installer/action_lib/__init__.py

```diff
@@ -340,10 +340,15 @@ def _add_prereqs_script(chain):
 sudo tee /etc/systemd/system/docker.service.d/override.conf <<- EOF
 [Service]
 ExecStart=
-ExecStart=/usr/bin/docker daemon --storage-driver=overlay -H fd://
+ExecStart=/usr/bin/docker daemon --storage-driver=overlay -H unix:///var/run/docker.sock
 EOF
 
-sudo yum install -y docker-engine-1.11.2
+sudo systemctl daemon-reload
+# try to stop the older docker version, but do not hard fail if it does not
+# exist.
+sudo systemctl stop docker || true
```

Comment:
If we're trying to get rid of old dockers, we should probably try to stop docker.socket and docker.service?